### DISCUSSION TITLE
Wrong Approach YMHB STAR

### DISCUSSION THREAD

**Discussion by TVOpium** [flybywiresim/aircraft#3763] [General]
Hello,

I notice that on several airports there is a problem with the STAR.
When I select this arrival in fact the FMC does not follow the card procedure at all. Is this a known issue? Problem with FS2020 or Navigraph data?

<a href="https://ibb.co/DLbLR6Z"><img src="https://i.ibb.co/YbWbLKx/9e0e9d2cec1b61b3b23589364bc501135a351259.jpg" alt="9e0e9d2cec1b61b3b23589364bc501135a351259" border="0"></a>
<a href="https://imgbb.com/"><img src="https://i.ibb.co/yRKxD9c/ccff155a5898e2559fe0a1c8f9d6ef6b8b97f9b5.jpg" alt="ccff155a5898e2559fe0a1c8f9d6ef6b8b97f9b5" border="0"></a>

Sim: 1.13.0
A / C: A320 DEV Version
AIRAC: 2101 REV1
Airport and PROC : YMHB MORGO1A RNP

**Comment by hotshotp** [comment]
known issue, custom autopilot is in works

**Reply by robinottawa** [reply]
I know this kind of question comes up a lot, but "known issue"? Does that mean STARs is not usable until the custom AP comes out? That there's no help for this question or others like it?

**Comment by robinottawa** [comment]
So to be sure I understand (so I can compare notes), you enter a STAR for a certain arrival and you get neither of the 2 chart options but you do get a third option that is not on a chart? Can you fly the STAR?

EDIT: I'm just trying a STAR of my own and notice that cleaning up the route in the MSFS globe FP area does not help. I ended up with a bogus set of waypoints in my FMC. So I just re-entered the STAR in the MCDU and it cleaned right up. Have you tried that?

**Comment by TVOpium** [comment]
Richard (Navigraph Development Team member) replied to me and he said that with the CJ4 Mod the SID and STAR were correctly interpreted because this mod has the particularity of loading Navigraph data, but not the 32NX for the moment, I hope to quickly see this functionality brought. So there is also an Asobo limitation, and why it is complex. Same yesterday at the start of MKJP, impossible to make a correct SID no big deal I did it at the VOR. Have a nice day and thank you for your work and follow up guys!

**Reply by robinottawa** [reply]
Thanks. Good to know.

**Reply by TVOpium** [reply]
Hi Edouard,
that´s a known and from ASOBO/MS confirmed limitation in the sim. The FBW A320 uses still the internal logic of the flight-planning and that´s the result. This is also independent of the data you use (stock data from NavBlue or our data from Jeppesen). Sorry, we can only wait on a fix from ASOBO/MS because we can only offer the data but we have no influence of the interpretation of it … Also, this happens for ALL in-game aircrafts because all uses the internal logic of flight planning of the sim.

But what I can highly recommand is to try the Working-Title CJ4. The devs have developed a own, complete independent flight-planning for their FMC and here you see, how it will be worked, when all will be interpreted correctly and when the linkage between STARs and approaches are working.

Here the MORGO 1A to RWY 30 into Hobart:
![ba64164aeeb6d14293b4444ba6acc1c232e93131](https://user-images.githubusercontent.com/78448492/110154565-b2295200-7de4-11eb-89a9-e06a9634de18.jpeg)

You see, the same data as the sim uses but a perfect STAR with the VOR approach. No truncated waypoints, no strange turns or similar else. It´s really worth to try. The FBW guys are also working on a independent flight-planner for their FMC and I´m pretty sure, that then we have a good basic package.

Sorry, but the in-game aircrafts (nor the route-planning in the world-map) are very useful for IFR.

Cheers,
Richard

**Comment by TVOpium** [comment]
SID MLY7 RWY 12 MKJP

<a href="https://ibb.co/SxMk43p"><img src="https://i.ibb.co/n0YXNLS/Microsoft-Flight-Simulator-Screenshot-2021-03-05-09-37-12-18.png" alt="Microsoft-Flight-Simulator-Screenshot-2021-03-05-09-37-12-18" border="0"></a>

<a href="https://ibb.co/s3KCWP8"><img src="https://i.ibb.co/LpxrNPV/Sans-titre.png" alt="Sans-titre" border="0"></a>

**Reply by robinottawa** [reply]
(Hey, you might want to turn up the brightness on your panels. They're very hard to read. Dials are beside the MCDU and there's a toggle on the MCDU. cheers)

**Reply by TVOpium** [reply]
yes i know i did that quickly this morning ;)

**Comment by TVOpium** [comment]
On STAR at MWCR (VOR RWY26) i don't have a TRANS NALRO, i have on FMC (ATUVI, DELKA, EMONA, KANEX)

<a href="https://ibb.co/qCygs3x"><img src="https://i.ibb.co/3vrFN9k/Sans-titre.png" alt="Sans-titre" border="0"></a>

<a href="https://ibb.co/mSn269n"><img src="https://i.ibb.co/8Xyf6ry/Microsoft-Flight-Simulator-Screenshot-2021-03-05-09-41-33-92.png" alt="Microsoft-Flight-Simulator-Screenshot-2021-03-05-09-41-33-92" border="0"></a>
